decaf377-fmd/src/clue_key.rs

```diff
@@ -21,8 +21,9 @@ pub struct ClueKey(pub [u8; 32]);
 /// intended for the corresponding [`DetectionKey`](crate::DetectionKey).
 #[allow(non_snake_case)]
```

Comment:
nit: now that the `Xs` field is gone, the `#[allow(non_snake_case)]` could be removed too